steps/previsit/index.ts

```diff
@@ -1,5 +1,6 @@
+import aboutVisit from "./about-visit";
+import firstAttempt from "./first-attempt";
 import outside from "./outside";
 import start from "./start";
-import aboutVisit from "./about-visit";
 
-export default [outside, start, aboutVisit];
+export default [outside, start, aboutVisit, firstAttempt];
```

Comment:
Do we consider this "previsit"? Should we pull it out into its own folder `unable-to-enter`?